### DISCUSSION TITLE
Which are the best hosting provider for OrchardCore?

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#8377] [General]
Which are the hosting providers which offer the best support and performance in ASP.NET Core and OrchardCore?

**Comment by Skrypt** [comment]
Microsoft Azure. For best support of features. Performance, I can't say, never compared. Though it's way fast enough for us.

**Reply by DotCat1985** [reply]
Which components should I use? I need to deploy a multilingual website with 8 pages, including a contact form.

**Reply by Skrypt** [reply]
If you don't need more than this then any host would be fine. Though, if you want to use Azure Blob Storage for storing medias, config and else it's quite usefull to be on Azure because it just works.

Content Localization, Localization, Forms, Flow Part features.
